scripts/services/rewrite-services-json.js

```diff
@@ -0,0 +1,78 @@
+const { promises: fs } = require('fs');
+const path = require('path');
+const yaml = require('js-yaml');
+
+const { checkSVG } = require('./check-svg');
+
+/**
+ * Checks that all services have valid SVG icons.
+ *
+ * @param {Array<object>} servicesArray - An array of service data objects.
+ * @returns {boolean} True if all services have valid SVG icons.
+ * @throws {Error} If an error(s) occurred during SVG validation, an error is thrown.
+ */
+const validateSvgIcons = (servicesArray) => {
+    // Array with results of svg validation.
+    const svgValidationResults = servicesArray.map((service) => checkSVG(service.icon_svg, service.id));
+    // Flatten array and filter nullable values.
+    const errorReports = svgValidationResults.flat().filter((el) => el);
```

Comment:
map (above), flat and filter — too many array iterations

error can be collected in this way:
```
const errorReports = [];

servicesArray.forEach((s) => {
    const errors = checkSVG(service.icon_svg, service.id);
    if (errors.length > 0) {
        errorReports.push(...strings);
    }
});
```